pkgs/development/python-modules/black/default.nix

```diff
@@ -23,14 +23,19 @@ buildPythonPackage rec {
 
   # Don't know why these tests fails
   # Disable test_expression_diff, because it fails on darwin
+  # Disable Primer* tests because they depend on black beeing in PATH
   checkPhase = ''
     LC_ALL="en_US.UTF-8" pytest \
       --deselect tests/test_black.py::BlackTestCase::test_expression_diff \
       --deselect tests/test_black.py::BlackTestCase::test_cache_multiple_files \
       --deselect tests/test_black.py::BlackTestCase::test_failed_formatting_does_not_get_cached
```

Comment:
@TheDelus Missing a continuation backslash in the line above.

Otherwise LGTM.

Reply:
Done. Thank you!